### ISSUE TITLE
gatsby-plugin-image object-fit is not reflected on placeholder

### ISSUE THREAD

**Issue by F1ow** [gatsbyjs/gatsby#30841]
## Description

Using `<GatsbyImage image={image} objectFit="contain" />` does not set the correct styles for the placeholder image.

### Steps to reproduce

https://github.com/F1ow/gatsby-plugin-image-object-fit-issue
1. Clone Repo
2. Run
3. Inspect Image

### Expected result

Placeholder has correct `object-fit` styles.
`<img aria-hidden="true" data-placeholder-image="" decoding="async" src="" alt="" style="transition: opacity 500ms linear 0s; object-fit: contain;">`

### Actual result

Placeholder has still `object-fit: cover;`
`<img aria-hidden="true" data-placeholder-image="" decoding="async" src="" alt="" style="transition: opacity 500ms linear 0s;">`

### Environment

```
  System:
    OS: macOS 11.2.3
    CPU: (12) x64 Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 14.15.4 - /usr/local/bin/node
    Yarn: 1.22.10 - /usr/local/bin/yarn
    npm: 7.5.0 - /usr/local/bin/npm
  Languages:
    Python: 2.7.16 - /usr/bin/python
  Browsers:
    Chrome: 89.0.4389.114
    Firefox: 87.0
    Safari: 14.0.3
  npmPackages:
    gatsby: ^3.2.1 => 3.2.1 
    gatsby-plugin-image: ^1.2.1 => 1.2.1 
    gatsby-plugin-sharp: ^3.2.1 => 3.2.1 
    gatsby-source-filesystem: ^3.2.0 => 3.2.0 
    gatsby-transformer-sharp: ^3.2.0 => 3.2.0 
```

**Comment by ascorbic** [comment]
Thanks for the report. I've opened a PR with a fix

**Comment by ascorbic** [comment]
This should be fixed in `gatsby-plugin-image@1.4.0-next.1`
